lib/feedback.mli

```diff
@@ -15,6 +15,7 @@ type level =
   | Debug
   | Info
   | Notice
+  | Prompt
```

Comment:
Umm, is this supposed to stay? I'm not sure this is the right place as feedback is usually attached to sentences and plays a quite different role.

Reply:
"Notice" appends a newline to every message.  It seems lame to put a newline after the debugger prompt "TcDebug > ".  I don't like the idea of scanning the output in CoqIDE and suppressing the newline when it matches the debug prompt.  How would you handle it?

BTW, if we create a "Debug" panel separate from the Message panel, then IIUC from @gares, I would need a way to pass a "route id" so the output ends up there.  That might mean adding a "Debugger" category or, maybe better, a labelled "route" parameter on `Feeback.msg_notice` et al.

Reply:
Feedback shouldn't be used for the debug channel, I think, as it corresponds more to the notion of diagnostic found in common UI protocols.